components/esp_eth/src/esp_eth_phy_ksz80xx.c

```diff
@@ -128,6 +149,10 @@ static esp_err_t ksz80xx_update_link_duplex_speed(phy_ksz80xx_t *ksz80xx)
                 ksz8041_pc2r_reg_t pc2r;
                 ESP_GOTO_ON_ERROR(eth->phy_reg_read(eth, ksz80xx->addr, KSZ8041_PC2R_REG_ADDR, &(pc2r.val)), err, TAG, "read PC2R failed");
                 op_mode = pc2r.op_mode;
+            } else if (ksz80xx->vendor_model == KSZ8061_MODEL_ID) {
```

Comment:
If the KSZ8061 PC1R register was the exactly the same as for KSZ8081, we could simplify the code - remove this `if` statement and the extend the below one as follows:
``` c
else if (ksz80xx->vendor_model == KSZ8081_MODEL_ID || ksz80xx->vendor_model == KSZ8061_MODEL_ID) {
```

Reply:
Well, I can do that. But then, it is also neccessary to find a better name for the type `ksz8081_pc1r_reg_t` as it represents more than the ksz8081.

Otherwise, the code readability would breake (in my opinion), as at the first sight it seams to be a bug if using ksz8081_pc1r_reg_t register inside the if block also checking against the ksz8061 model.

Reply:
Yeah, we need to find some appropriate name. Could you please investigate what is the difference between ksz8081 and ksz8061 at first?

Reply:
Not neccessary anymore with the reworked implemetation